workflow/README.md

```diff
@@ -174,13 +241,13 @@ type Workflow struct {
 
 The global defaults object contains options for the Workflow itself, as well as cascading defaults for Tasks.
 
-| Field    | Type              | Description                             | Required | Default         |
-|----------|-------------------|-----------------------------------------|----------|-----------------|
-| `secret` | `Boolean or null` | Whether the output is unsafe to publish | No       | `null`          |
-| `check`  | `Verification`    | How to verify the output                | No       | `"attestation"` |
-| `time`   | Integer           | Timeout in milliseconds                 | No       | `5000`          |
-| `memory` | Integer           | Memory limit in KB                      | No       | `1000`          |
-| `disk`   | Integer           | Disk limit in KB                        | No       | `100000`        |
+| Field    | Type              | Description                             | Required | Default                  |
+|----------|-------------------|-----------------------------------------|----------|--------------------------|
+| `secret` | `Boolean or null` | Whether the output is unsafe to publish | No       | `null`                   |
+| `check`  | `Verification`    | How to verify the output                | No       | `"attestation"`          |
+| `time`   | `TimeLength`      | Timeout                                 | No       | `[5, "minutes"]`         |
+| `memory` | `InfoSize`        | Memory limit                            | No       | `[100, "kilo", "bytes"]` |
+| `disk`   | `InfoSize`        | Disk limit                              | No       | `[10, "mega", "bytes"]`  |
```

Comment:
Not sure if you are intending this struct to be used by more than IPVM, but if yes, we also have things like number of GPUs. If you do want this to be standard, I'll pull out all of our other config options.